gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/resources/groups/GroupEndpoint.java

```diff
@@ -119,6 +120,64 @@ public void update(RoutingContext context) {
         }
     }
 
+    /**
+     * HTTP PATCH is an OPTIONAL server function that enables clients to
+     * update one or more attributes of a SCIM resource using a sequence of
+     * operations to "add", "remove", or "replace" values.  Clients may
+     * discover service provider support for PATCH by querying the service
+     * provider configuration (see Section 4).
+     *
+     * The general form of the SCIM PATCH request is based on JSON Patch
+     * [RFC6902].  One difference between SCIM PATCH and JSON Patch is that
+     * SCIM servers do not support array indexing and do not support
+     * [RFC6902] operation types relating to array element manipulation,
+     * such as "move".
+     *
+     * The body of each request MUST contain the "schemas" attribute with
+     * the URI value of "urn:ietf:params:scim:api:messages:2.0:PatchOp".
+     *
+     * The body of an HTTP PATCH request MUST contain the attribute
+     * "Operations", whose value is an array of one or more PATCH
+     * operations.  Each PATCH operation object MUST have exactly one "op"
+     * member, whose value indicates the operation to perform and MAY be one
+     * of "add", "remove", or "replace".  The semantics of each operation
+     * are defined in the following subsections.
+     *
+     * See <a href="https://tools.ietf.org/html/rfc7644#section-3.5.2">3.5.2.  Modifying with PATCH</a>
+     */
+    public void patch(RoutingContext context) {
+        try {
+            final PatchOp patchOp = Json.decodeValue(context.getBodyAsString(), PatchOp.class);
+            final String groupId = context.request().getParam("id");
+
+            // schemas field is REQUIRED and MUST contain valid values and MUST not contain duplicate values
+            try {
+                checkSchemas(patchOp.getSchemas(), PatchOp.SCHEMAS);
+            } catch (Exception ex) {
+                context.fail(ex);
+                return;
+            }
+
+            // check operations
+            if (patchOp.getOperations() == null || patchOp.getOperations().isEmpty()) {
+                throw new InvalidValueException("Field [Operations] is required");
```

Comment:
why throwing an exception here instead calling context.fail() and then return ?

Reply:
Done.